src/udisksdaemon.c

```diff
@@ -1804,13 +1809,17 @@ udisks_daemon_get_parent_for_tracking (UDisksDaemon  *daemon,
 
   UDisksObject *object = NULL;
   UDisksObject *crypto_object = NULL;
+#ifdef HAVE_MDRAID
```

Comment:
Some of the ifdefs are not necessary, for the sake of code cleaniness - you may have an object declared, assigned to NULL and unused - that's perfectly fine.